xml/decoders/lenz/accelDecel.xml

```diff
@@ -55,6 +55,8 @@
     <comment xml:lang="de">Wertebereich 0-255</comment>
     <label xml:lang="cs">Zrychlení</label>
     <comment xml:lang="cs">Meze 0-255</comment>
+    <label xml:lang="cs">Inèrcia d'Acceleració</label>
+    <comment xml:lang="cs">Rang de 0-255</comment>
```

Comment:
The XML file is not allowed to have two 'label' elements for the same language, so this is an error because there's already a 'label' with `xml:lang="cs"`. Perhaps this was meant as a replacement for that line?

Same for two 'comment' elements in the same language.